pytest_xmlcomp/plugin.py

```diff
@@ -100,14 +100,14 @@ def collect(self):
             print("JSON Syntax Error in file %s:\n%s" % (jsonfile, error),
                   file=sys.stderr)
             return
-        for xpath, expresult in jsondata:
-            yield XPathItem(xpath, self, expresult, tree)
+        namesp = None if not jsondata.get('ns') else dict(jsondata.get('ns'))
```

Comment:
Do we need the `dict()` around `jsondata.get` in the last expression? Isn't a dict already returned?